runtime/compiler/runtime/JitRuntime.cpp

```diff
@@ -1330,13 +1330,23 @@ uint32_t isRecompMethBody(void *li)
 
 // This method MUST be used only for methods that were AOTed and then relocated
 // It marks the BodyInfo that this is an aoted method.
-void fixPersistentMethodInfo(void *table)
+void fixPersistentMethodInfo(void *table, bool isJITServer)
```

Comment:
It occurred to me that, based on the comment, this method will be executed for relocations which are only done at the client. 
If you decide to leave it like this you should open a cleanup issue.

Reply:
Ran some tests. Looks like `fixPersistentMethodInfo()` is only called at the client. However, it can be called at the client for both `isAOTLoad=true` and `isAOTLoad=false` cases in `TR_RelocationRecordBodyInfo::applyRelocation()`. I'd propose to keep the current change but rename the parameter `isJITServer` to `isAOTLoad` to better reflect the condition check.

Reply:
Will go with `isJITClientAOTLoad` as discussed.